### DISCUSSION TITLE
Font in Thematic Guides' "FontsExample" is not valid

### DISCUSSION THREAD

**Discussion by yasenstar** [ManimCommunity/manim#4047] [General] [answered]
Hello all,

Learning the Thematic Guides, the sample: https://docs.manim.community/en/stable/guides/using_text.html#using-fonts which is Using Fonts pop up warning that "Noto Sans" is not available.

Also if you check the complete font list, this "Noto Sans" is not existed as well.

Could you please check and if that's the case, butter the correct the online sample code.

Thanks,
Xiaoqi

**Comment by behackl** [comment]
Hello! Where exactly do you see this warning pop up?

In principle, you can pass all fonts that are installed locally on your machine; if Noto Sans is not available, you'll need to install it first.

**Reply by yasenstar** [reply]
Hello,

See this screenshot attached, it's in online with the Docker container executing.

Xiaoqi

![manim-use-fonts-missing](https://github.com/user-attachments/assets/a2406c25-cf63-432f-a819-a22467bc0b0c)

**Reply by behackl** [reply, marked as answer]
Ah, I see! You do mean in the interactive container -- this is something we can fix. Thanks for letting us know!

**Reply by yasenstar** [reply]
Thanks for your quick confirmation.
